### ISSUE TITLE
Zone Overhaul - Tirisfal Glades - MinLevel / QuestLevel

### ISSUE THREAD

**Issue by Tobschinski** [classicdb/database#304]
Some 'minlevel / level' values in Tirisfal Glades (Zone: 85) may be wrong, regarding Bradygames' Master Guide, web.archive.org/2006/wow.allakhazam.com and web.archive.org/2006/wowhead.com.

**Comment by Tobschinski** [comment]
```
UPDATE `quest_template` SET `MinLevel`='5' WHERE `entry`='366';
UPDATE `quest_template` SET `MinLevel`='6' WHERE `entry`='369';
UPDATE `quest_template` SET `MinLevel`='5' WHERE `entry`='371';
UPDATE `quest_template` SET `MinLevel`='5' WHERE `entry`='372';
UPDATE `quest_template` SET `MinLevel`='6' WHERE `entry`='398';
UPDATE `quest_template` SET `MinLevel`='7' WHERE `entry`='408';
UPDATE `quest_template` SET `MinLevel`='5' WHERE `entry`='409';
UPDATE `quest_template` SET `MinLevel`='5' WHERE `entry`='410';
UPDATE `quest_template` SET `MinLevel`='5' WHERE `entry`='411';
UPDATE `quest_template` SET `MinLevel`='5' WHERE `entry`='431';
UPDATE `quest_template` SET `MinLevel`='6' WHERE `entry`='492';
```

**Comment by Tobschinski** [comment]
@cala - If you want me to merge all these issues to not lose track, please let me know. : /

**Comment by cala** [comment]
That is a lot of issues, indeed but I think keeping them separated by zone/area is a good thing. However, the level, racial, reputation could be regrouped into the same issue (still by zone) as they are all fixed the same table `quest_template`. Thanks for your work.

**Comment by cala** [comment]
Also, I noticed that you always wrote:

> Some 'XXX' values in XXXX (Zone: XX) **may** be wrong

Unless I'm mistaken, I understand you are not sure that the values are wrong. Whenever possible, please only open issues that are confirmed:
- there were a lot of issues opened the last few weeks (which is nice) but I'm starting to lose feet on them and as such it is difficult to priorise them
- before fixing an issue, I always look at at least two sources (when available) in order to check the validity of the issue/fix. If it happens there is nothing to fix, then time was spent on something when it would have been more useful elsewhere

**Comment by Tobschinski** [comment]
Well, I know for sure, most experience-values are wrong (~85%), simply because zones got overhauled during TBC and WotLK. 
Additionally, many EP-values in ManGOS-DB have been rounded.

But also, I can never be sure, my provided values are 100% correct.
This is simply to explain with the fact, that websites were missing data in vanilla and I do not have any info about the authors of Bradygames' Master Guide (BMG), released in October 2006. So I can only assume they have correct values.

minLVL / qLVL is way easier to prove, current wowhead.com matches BMG  at 99%.
(again, wowhead.com may be imprecise, because some quests had a higher level when they have been tagged elite, which has been changed during later expansions)

---

But I do have to apologize to put 'may' in false position and even copy-pasted it into every single issue.

Values ARE incorrect and my provided fixes MAY be the correct ones. (I will always say 'may', even if I am 99.99% sure, there is still a chance I am wrong)

**Comment by Tobschinski** [comment]
https://github.com/classicdb/database/issues/306

**Comment by Tobschinski** [comment]
I'll give you a quick example to understand how 'corrupted' current values are.

Please, check your database for quest-name "A Plague upon Thee". You will find 6 quests, 3 for Horde, 3 for Alliance.

#6389 and #6390 have a different 'MinLevel', although it is the same quest, for different factions. That does not make any sense and is pretty much a mistake. (Bradygames' Guide and wowhead are sure about that)

Now check maximum experience. You will see 5 different values, although there are only 3 different quests. Granting less experience to Horde or Alliance for the same quest, is again, pretty much a mistake.

If you check this [video](https://www.youtube.com/watch?feature=player_detailpage&v=PCO8IIz9YHA#t=23) you will see, the second part of this quest should grant a maximum of 2800 ep. That matches Bradygames, but not the current value.

Sadly, mistakes like that occur almost everywhere.

**Comment by cala** [comment]
> But I do have to apologize to put 'may' in false position and even copy-pasted it into every single issue.

No worries, as english is not my native language, I prefer to double check that I understand what others say/write. ;)

> Granting less experience to Horde or Alliance for the same quest, is again, pretty much a mistake.

I'm not sure that is a mistake. I mean, Horde have cows as a playable race. They should have a penalty for that. Well... On the other hand, Alliance have gnomes... Let's call it a tie game, ok? ;)

> Well, I know for sure, most experience-values are wrong (~85%), simply because zones got overhauled during TBC and WotLK. 

Yes, the leveling curve was lowered a first time in patch 2.3 (I remember thinking at the time how it was a mistake to devalue old content instead of drawing interest into it) and a second time in WotLK. Thus there are good chances that values in the current DB may be wrong as many of them are backports from TBC-DB/UDB. However, not all quests were changed: I remember fixing some quests values that were wrong in the DB (obviously a typo), but official WotLK values were still unchanged since 2006 based on sources.

> If you check this video you will see, the second part of this quest should grant a maximum of 2800 ep. That matches Bradygames, but not the current value.

I agree Bradygames books are pretty reliable sources, probably more than wowhead/wowwiki/ thotbot/allakazam... I'd still be careful as I had a Bradygames strategy guide for Diablo II and there were some mistakes in it. It is due to the fact that Blizzard indeed gives real data to Bradygames, but in order to have the book printed and available when game is released, the data are pre-release values and some of them are changed in the few last QA tests rounds. However, regarding WoW, I think it would be more the creature stats that were changed rather than the XP values.

So to summarize, I understand that the values you've pointed out have very good chance to be wrong and you intend to ckeck them against Bradygames / archived sources. Sounds good to me. :+1:

**Comment by Tobschinski** [comment]
Oh, I know your feels ...
My first char was a gnomish mage, I didn't know anything about WoW. 'till level 10 I did nothing but punching NPCs faces in with my staff. I did not use any spells, I thought they are scrolls and I wanted to save them. (in my defence, I've played Gothic 1 & 2 before)

At some point I raged and deleted him, as leveling was a pain.
I have never recovered from that and since then, I f**king hate gnomes.

**Comment by Tobschinski** [comment]
By the way, you have gotten me even more into Bradygames' Guides. I didn't know they have taken their data directly from Blizzard.

And since the Master Guide (October 2006) is the overhauled edition of the Official Strategy Guide (~2005), I do have hope they have taken care of typos or value-changes.
